### DISCUSSION TITLE
Should the nullpomino cask use the jar instead of dmg so it has apple silicon support

### DISCUSSION THREAD

**Discussion by superusercode** [Homebrew/discussions#4591] [Casks] [answered]
### Output of `brew config`

```shell
not relevant
```

### Output of `brew doctor`

```shell
not relevant
```

### Description of issue

nullpomino is a java app and can run on apple silicon without rosetta, but the dmg is from before apple silicon released and the program is no longer actively maintained. It's also unlikely a new apple silicon version can be released in place since my experience with the packaging tech it used was not maintained when I initially explored building nullpomino and its codebase. 

### Relevant casks

nullpomino

**Comment by SMillerDev** [comment, marked as answer]
Sounds reasonable, please make a pull request.
